src/containers/network/AboutNetwork.js

```diff
@@ -1,3 +1,17 @@
 import React from 'react'
-const AboutNetwork = props => <span />
-export default AboutNetwork
+import { connect } from 'react-redux'
+import { markdown } from '../../util/text'
+const { object } = React.PropTypes
+
+@connect((state, { params }) => ({network: state.networks[params.id]}))
+export default class AboutNetworks extends React.Component {
```

Comment:
oh i wrote all that before i noticed you'd already done it for `NetworkPosts` :)